op-deployer/pkg/deployer/manage/testdata/state.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": 1,
+  "create2Salt": "0xe67da0752add941001738a7742b5137ccc4a96913a56bc31a037e4c8244d74ac",
+  "appliedIntent": {
+    "configType": "standard",
+    "l1ChainID": 11155111,
+    "opcmAddress": "0xfBceeD4DE885645fBdED164910E10F52fEBFAB35",
+    "superchainRoles": {
+      "proxyAdminOwner": "0x1eb2ffc903729a0f03966b917003800b145f56e2",
+      "protocolVersionsOwner": "0xfd1d2e729ae8eee2e146c033bf4400fe75284301",
+      "guardian": "0x7a50f00e8d05b95f98fe38d8bee366a7324dcf7e"
+    },
+    "fundDevAccounts": false,
+    "useInterop": false,
+    "l1ContractsLocator": "tag://op-contracts/v3.0.0-rc.2",
+    "l2ContractsLocator": "tag://op-contracts/v3.0.0-rc.2",
+    "chains": [
+      {
+        "id": "0x00000000000000000000000000000000000000000000000000000000000004d2",
+        "baseFeeVaultRecipient": "0x0000000000000000000000000000000000000001",
+        "l1FeeVaultRecipient": "0x0000000000000000000000000000000000000002",
+        "sequencerFeeVaultRecipient": "0x0000000000000000000000000000000000000003",
+        "eip1559DenominatorCanyon": 250,
+        "eip1559Denominator": 50,
+        "eip1559Elasticity": 6,
+        "roles": {
+          "l1ProxyAdminOwner": "0x1eb2ffc903729a0f03966b917003800b145f56e2",
+          "l2ProxyAdminOwner": "0x2fc3ffc903729a0f03966b917003800b145f67f3",
+          "systemConfigOwner": "0x0000000000000000000000000000000000000004",
+          "unsafeBlockSigner": "0x0000000000000000000000000000000000000005",
+          "batcher": "0x0000000000000000000000000000000000000006",
+          "proposer": "0x0000000000000000000000000000000000000007",
+          "challenger": "0xfd1d2e729ae8eee2e146c033bf4400fe75284301"
+        }
+      }
+    ],
+    "globalDeployOverrides": null
+  },
+  "superchainDeployment": {
+    "proxyAdminAddress": "0x189abaaaa82dfc015a588a7dbad6f13b1d3485bc",
+    "superchainConfigProxyAddress": "0xc2be75506d5724086deb7245bd260cc9753911be",
+    "superchainConfigImplAddress": "0x0000000000000000000000000000000000000000",
+    "protocolVersionsProxyAddress": "0x79add5713b383daa0a138d3c4780c7a1804a8090",
+    "protocolVersionsImplAddress": "0x0000000000000000000000000000000000000000"
+  },
+  "implementationsDeployment": {
+    "opcmAddress": "0xfbceed4de885645fbded164910e10f52febfab35"
+  },
```

Comment:
Can delete these since they're unused for testing and also the keys are incompatible with current version of state file